app/controllers/course/enrol_requests_controller.rb

```diff
@@ -0,0 +1,128 @@
+class Course::EnrolRequestsController < Course::ModuleController
+  load_and_authorize_resource through: :course
+  skip_before_filter :verify_authenticity_token, only: [:approve_selected, :delete_selected]
+
+  def index #:nodoc:
+    @staff_requests = @enrol_requests.staff
+    @student_requests = @enrol_requests.student
+
+    respond_to do |format|
+      format.html
+    end
+  end
+
+  def new #:nodoc:
```

Comment:
Assignment Branch Condition size for new is too high. [25.57/15]<br>Method has too many lines. [22/10]